apps/timetable/admin/index.html

```diff
@@ -154,6 +154,14 @@
             }) %>
         </script>
 
+        <!-- Delete series modal -->
+        <div id="gh-delete-series-modal-container"><!-- --></div>
+        <script id="gh-delete-series-modal-template" type="text/template">
+            <%= _.partial('delete-series-modal', {
+                'data': data
+            }) %>
```

Comment:
Shall we use a one-liner for this, since we're only using `data` as a parameter here? WDYT?

`<%= _.partial('delete-series-modal', {'data': data}) %>`